java-rest-service/accelerator.yaml

```diff
@@ -80,8 +80,25 @@ accelerator:
       defaultValue: "Manage customer profiles"
       dependsOn:
         name: exposeOpenAPIEndpoint
+    - name: updateBoot3
+      inputType: checkbox
+      dataType: boolean
+      label: Use Spring Boot 3.0 (Requires Java 17)
+      defaultValue: false
+    # override java version choices, Java 8 is not available for this sample
+    - name: javaVersion
```

Comment:
Since Spring Boot 3 requires a Java 17 baseline, we can probably make the `javaVersion` option depend on `updateBoot3` being `false`.

Reply:
good idea, will try that